templates/shared_services/gitea/porter.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 schemaVersion: 1.0.0
 name: tre-shared-service-gitea
-version: 0.6.10
+version: 0.6.11
```

Comment:
As per https://github.com/microsoft/AzureTRE/issues/3540#issuecomment-1985330317, all version number should be major version upgrades as upgrade of mysql will not work.